docs/en/docs/admin-manual/privilege-ldap/user-privilege.md

```diff
@@ -236,9 +236,9 @@ ADMIN_PRIV and GRANT_PRIV have the authority of **"grant authority"** at the sam
 
 5. Forget passwords
 
-	If you forget your password and cannot log in to Doris, you can log in to Doris without a password using the following command on the machine where the Doris FE node is located:
+	If you forget your password and cannot log in to Doris, you can add `skip_localhost_auth_check` in fe config so that logging to Doris without a password in localhost.
```

Comment:
Do we need to restart FE? If yes, please add it in doc

Reply:
Need restart FE